twarep/electrifly
# Scraper run log enhancement

## Checklist

## What I Did
What files git you edit and why/for what purpose?

**app.py**: replaced function to read last scraper run from txt file to the database table
**flight_querying.py**: added function for app.py to use to get the most recent runtime
**queries.py**: added query to create the runtime table
**scraper.py**: updated the function to read last scraper run from txt file to the database table
**storage.py**: added script to delete the old run and push the newest runtime to the database

## How to Test?
Manual testing? Automated testing? E2E testing? Smoke tests?

Run scraper.py, note the time you ran it, then run app.py and check data preview to see if the most recent runtime is equal to that time.
